config-store-yaml/README.adoc

```diff
@@ -0,0 +1,74 @@
+= Using Config Store Yaml
+
+This extension provides support for loading stargate configuration from a YAML file.
+
+== YAML File Location
+
+When the `ConfigStoreYaml` is activated (registered in the OSGi), it will try to lookup the stargate config
```

Comment:
To me it looks like `activated` and `registered` are conflicting in this context... `registered` is also mentioned below as an _optional_ event :thinking:

Reply:
right, I've unified it